src/ir/function.rs

```diff
@@ -131,7 +131,8 @@ pub fn cursor_mangling(ctx: &BindgenContext,
     }
 
     // Try to undo backend linkage munging (prepended _, generally)
-    if cfg!(target_os = "macos") {
+    if get_abi(cursor.cur_type().call_conv()).map_or(true, |abi| abi != abi::Abi::Stdcall) &&
```

Comment:
This only seems to be working around a test expectation difference on OSX while running with `-target msvc`, but this is not a proper fix.

The difference happen because this `cfg!` check is on the target that compiled bindgen, not on the target that clang is parsing files as. That's a totally unrelated fix, and you should just skip adding that file to the diff and revert this change.